stack.yaml

```diff
@@ -96,4 +100,6 @@ extra-deps:
 # compiler-check: newer-minor
 
 nix:
+  enable: true
+  pure: false
```

Comment:
It should be pure if all the dependencies are provided from the Nix environment, right?

Reply:
right, currently git is not provided. It cannot be added easily until we get input-output-hk/haskell.nix#121.